src/04-modality-specific-files/29-motion.md

```diff
@@ -0,0 +1,224 @@
+
+Note that the date and time information SHOULD be stored in the Study key file [(`scans.tsv`)](https://bids-specification.readthedocs.io/en/stable/03-modality-agnostic-files.html#scans-file). Date time information MUST be expressed as indicated in Units. The [`scans.tsv`](https://bids-specification.readthedocs.io/en/stable/03-modality-agnostic-files.html#scans-file) file contains the filename and the acquisition time of a recording, which can be used to synchronize streams. However, synchronization information between the two systems can also be stored using time stamps in the `*_motion.tsv` if available?.
+
+## Channels description (`*_channels.tsv`)
+
+```markdown
+└─ sub-<label>\
+  └─ \[ses-<label>]\
+    └─ motion\
+      └─ sub-<label>\[\_ses-<label>][\_task-<label>\]_channels.tsv
+```
+
+This file is REQUIRED as it makes it easy to browse or query over larger collections of datasets. The REQUIRED columns are channel  `component`, `name`,  `tracked_point`, `type` and `unit`. Any number of additional columns may be added to provide additional information about the channels. The `*tracksys-<label>_channels.tsv` file should give additional information about individual recorded channel, some of which my not be found summarized in `*motion.json`.
+
+The columns of the channels description table stored in `*_channels.tsv` are:
+
+MUST be present:
+
+| **Key name**  | **Requirement level** | **Data type** | **Description**                                                                                                                                                                                |
+| ------------- | --------------------- | ------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| component     | REQUIRED              | string        | Component of the representational system described in `*_coordinatesystem.tsv` that the channel contains.                                                                                      |
+| name          | REQUIRED              | string        | Label of the channel. Entries have to match headers in (any) `*_motion.tsv.`                                                                                                                   |
+| tracked_point | REQUIRED              | string        | Label of the point that is being tracked, for example, label of a tracker or a marker (`LeftFoot`, `RightWrist`).                                                                              |
```

Comment:
Would this be related to the `BodyPart`, `BodyPartDetails`, `BodyPartDetailsOntology` in the Microscopy specs?

Reply:
It would I guess. The string label assigned by the user does not follow a strict schema as there are no standards in the community.